deploy/chart/templates/0000_30_13-packageserver.csv.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: packageserver.v8.0.0
+  namespace: {{ .Values.namespace }}
+spec:
+  displayName: Package Server
+  description: Package Server is kubernetes APIService that provides the PackageManifest resource
+  keywords: ['packagemanifests', 'olm', 'packages']
+  maintainers:
+  - name: Red Hat
+    email: openshift-operators@redhat.com
+  provider:
+    name: Red Hat
+  links:
+  - name: Package Server
+    url: https://github.com/operator-framework/operator-lifecycle-manager/tree/master/pkg/package-server
+  install:
+    strategy: deployment
+    spec:
+      deployments:
+      - name: package-server
+        spec:
+          replicas: {{ .Values.package.replicaCount }}
+          strategy:
+            type: RollingUpdate
+          selector:
+            matchLabels:
+              app: package-server
+          template:
+            metadata:
+              labels:
+                app: package-server
+            spec:
+              serviceAccountName: olm-operator-serviceaccount
+              containers:
+              - name: package-server
+                command:
+                - /bin/package-server
+                - -v=4
+                {{- if .Values.watchedNamespaces }}
+                - --watched-namespaces
+                - {{ .Values.watchedNamespaces }}
+                {{- end }}
+                - --secure-port={{ .Values.package.service.internalPort }}
+                - --global-namespace
+                - {{ .Values.namespace }}
+                {{- if .Values.debug }}
+                - --debug
+                {{- end }}
+                {{- if .Values.package.commandArgs }}
+                - {{ .Values.package.commandArgs }}
+                {{- end }}
+                image: {{ .Values.package.image.ref }}
+                imagePullPolicy: {{ .Values.package.image.pullPolicy }}
+                ports:
+                - containerPort: {{ .Values.package.service.internalPort }}
+                livenessProbe:
+                    httpGet:
+                      scheme: HTTPS
+                      path: /healthz
+                      port: {{ .Values.package.service.internalPort }}
+                readinessProbe:
+                    httpGet:
+                      scheme: HTTPS
+                      path: /healthz
+                      port: {{ .Values.package.service.internalPort }}
+  maturity: alpha
+  version: 8.0.0
+  apiservicedefinitions:
+    owned:
+    - group: packages.apps.redhat.com
+      version: v1alpha1
+      kind: PackageManifest
+      displayName: PackageManifest
+      description: A PackageManifest is a resource generated from existing CatalogSources and their ConfigMaps
```

Comment:
`Represents an Operator package that is available from a given CatalogSource which will resolve to a ClusterServiceVersion`.